easybuild/easyblocks/a/aocc.py

```diff
@@ -67,6 +67,10 @@ def __init__(self, *args, **kwargs):
 
         self.clangversion = self.cfg['clangversion']
 
+        # Starting from LLVM v16, the folder naming is based on the major version of clang.
+        if LooseVersion(self.clangversion) >= LooseVersion('16.0.0'):
+            self.clangversion = '16'
+
```

Comment:
This will break once AOCC updates to a newer Clang version. In addition, updating the `clangversion` to the major version only feels not correct. This is only needed for the sanity check. Therefore, such conversion should only be done where necessary. 

In the Clang EasyBlock, this is done. I've posted a draft for AOCC in the issue for this EasyBlock, but didn't have the time to create a PR and test it.